packages/redux/src/returns/__tests__/reducer.test.ts

```diff
@@ -249,6 +285,33 @@ describe('returns reducer', () => {
       ).toBe(initialState.isLoading);
     });
 
+    it('should handle FETCH_RETURN_PICKUP_RESCHEDULE_REQUEST_FAILURE action type', () => {
+      expect(
+        reducer(undefined, {
```

Comment:
For these tests, it is better to define a state where `isLoading` is different than `initialState.isLoading` to make sure the test will fail when the code does.

Reply:
done!